### PR TITLE
fix: reset accumulation in fracture when time step cut occurs in hydrofrac solver

### PR DESCRIPTION
**Pull request by Guotong-Ren**
When time step cut occurs, primary variables are reset to the beginning of newton, while the accumulation in fracture is not updated accordingly to pair with the initial guess.

### CODE REVIEW

**Review by CusiniM — APPROVED** [review]

**Review by paveltomin — APPROVED** [review]

File: src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp [hunk 1]
```diff
@@ -209,10 +209,10 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
     }
 
     // currently the only method is implicit time integration
-    dtReturn = nonlinearImplicitStep( time_n, dt, cycleNumber, domain );
+    dtReturn = nonlinearImplicitStep( time_n, dtReturn, cycleNumber, domain );
```
Comment (binw1) [inline src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: Simply change dt to dtReturn here may still have issues, as dt used in different fracture propagation iterations will still be different if timestep cut happens in some of the fracture propagation iteration.

File: src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp [hunk 2]
```diff
@@ -209,10 +209,10 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
...
 
 
-    if( !this->m_performStressInitialization && m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
+    if( !this->m_performStressInitialization && m_surfaceGenerator->solverStep( time_n, dtReturn, cycleNumber, domain ) > 0 )
```
Comment (binw1) [inline src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: Simply change dt to dtReturn here may still have issues, as dt used in different fracture propagation iterations will still be different if timestep cut happens in some of the fracture propagation iteration.

**Review by binw1 — APPROVED** [review]

File: src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp (hunk 2 above)
Comment (Guotong-Ren) [reply src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: yeah, for the time being, i reverted it back to original formulation. Thanks.

Comment (binw1) [reply src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: Yeah, I then saw the revert in the second commit

**Comment by Guotong-Ren** [comment]
@jhuang2601  can you please review this PR? Thanks!

**Review by jhuang2601 — APPROVED** [review]

**Comment by jhuang2601** [comment]
@Guotong-Ren You need to run the command in the the build folder: `make uncrustify_style` to fix the reported issue of CI failure on Check code style

**Pull request merged**